package.json

```diff
@@ -26,7 +26,7 @@
     "build:index": "webpack ./client-src/default/index.js -o client/index.bundle.js --color --config client-src/default/webpack.config.js",
     "build:live": "webpack ./client-src/live/index.js -o client/live.bundle.js --color --config client-src/live/webpack.config.js",
     "build:sockjs": "webpack ./client-src/sockjs/index.js -o client/sockjs.bundle.js --color --config client-src/sockjs/webpack.config.js",
-    "webpack-dev-server": "cd examples/api/simple && node ../../../bin/webpack-dev-server.js",
+    "webpack-dev-server": "cd $INIT_CWD && node ../../../bin/webpack-dev-server.js",
```

Comment:
Don't use this in package.json

Reply:
I returned it to `$INIT_CWD` I'm sorry I'm using windows I forgot to return it to $INIT_CWD in the last commit, now everything should work fine